libs/common/src/sql-db-entities/partial-entities/month-and-year.partial-entity.ts

```diff
@@ -1,6 +1,6 @@
 import { Column } from 'typeorm';
 
-export class MonthAndYearPartial {
+export class MonthAndYearPartialEntity {
```

Comment:
How about we keep these as `Partial` instead of `PartialEntity` and make the directory `partials`? Unlike `base-entities`, these do not exist as their own entities, so I think it may be better to remove the `Entity` label from them. What do you think?

Reply:
Not sure. I think you could make the same argument for base entities, as they should not exist on their own either, but are always extended.